starlite/plugins/tortoise_orm.py

```diff
@@ -86,29 +157,32 @@ def to_data_container_class(self, model_class: Type[Model], **kwargs: Any) -> Ty
         return self._models_map[model_class]
 
     @staticmethod
-    def is_plugin_supported_type(value: Any) -> "TypeGuard[Model]":
+    def is_plugin_supported_type(value: "Any") -> "TypeGuard[Model]":
         """Given a value of indeterminate type, determine if this value is supported by the plugin."""
         return isinstance(value, (Model, ModelMeta))
 
-    def from_data_container_instance(self, model_class: Type[Model], data_container_instance: "BaseModel") -> Model:
+    def from_data_container_instance(
+        self, model_class: "type[TortoiseModelT]", data_container_instance: "PydanticModel"
+    ) -> TortoiseModelT:
         """Given an instance of a pydantic model created using the plugin's ``to_data_container_class``, return an
         instance of the class from which that pydantic model has been created.
 
         This class is passed in as the ``model_class`` kwarg.
         """
         return model_class().update_from_dict(data_container_instance.dict())
 
-    async def to_dict(self, model_instance: Model) -> Dict[str, Any]:  # pylint: disable=invalid-overridden-method
+    def to_dict(self, model_instance: TortoiseModelT) -> "dict[str, Any]":
         """Given an instance of a model supported by the plugin, return a dictionary of serializable values."""
         pydantic_model_class = self.to_data_container_class(type(model_instance))
-        data = await pydantic_model_class.from_tortoise_orm(model_instance)
+        with anyio.start_blocking_portal() as portal:
```

Comment:
using a blocking portal here instead of making the protocol support both dict and awaitable return values for this one case.. but the tortoise tests end up hanging.. i've `pytest.mark.skip`ped them for now.

@starlite-api/maintainers anyone know how to handle this better (preferably modify the tests, assuming this is fine at runtime).

Reply:
You dont need the portal, you can do this:

```python
anyio.run(pydantic_model_class.from_tortoise_orm, model_instance)
```

Reply:
I started there - it was failing along the lines of there already being a running event loop.

Reply:
And

```python
from_thread.run_sync(...)
```

?

Reply:
I'll start again from the beginning and see if I was missing something obvious yesterday..